hibernate-rx-api/src/main/java/org/hibernate/rx/RxSession.java

```diff
@@ -84,7 +84,7 @@
 	/**
 	 * Asynchronously persist the given transient instance, first assigning
```

Comment:
Unless I'm missing something. Changes to the javadoc should be in a separate commit because they have nothing to do with the build.

Reply:
I can move them into a separate commit. They were related to publishing the javadoc artifacts because I had to make the javadoc HTML5 compatible

Reply:
Thanks, it's still nice to have them in a separate commit so that we can explain why that change is required.